tfhe/src/core_crypto/commons/mod.rs

```diff
@@ -336,4 +337,47 @@ pub mod test_tools {
         let failure_rate = failures / (RUNS as f64);
         assert!(failure_rate == 1.0);
     }
+
+    #[test]
+    pub fn test_torus_modular_diff() {
+        // q = 2^64
+        {
+            let q = CiphertextModulus::<u64>::new_native();
+            // Divide by 8 to get an exact division vs 10 or anything not a power of 2
+            let one_eigth = 1 << 61;
```

Comment:
It confused me at first, maybe adding a comment that (1<<61) is equal to 2^64 / 8 could make it more obvious for future readers

Reply:
good point

Reply:
I'll do a single edit for this once the tests pass